pkg/fleet-manager/plugin/plugin.go

```diff
@@ -32,25 +32,29 @@ import (
 )
 
 const (
-	MetricPluginName          = "metric"
-	GrafanaPluginName         = "grafana"
-	KyvernoPluginName         = "kyverno"
-	BackupPluginName          = "backup"
-	StorageOperatorPluginName = "storage-operator"
-	ClusterStoragePluginName  = "cluster-storage"
-	FlaggerPluginName         = "flagger"
-	PublicTestloaderName      = "testloader"
-
-	ThanosComponentName        = "thanos"
-	PrometheusComponentName    = "prometheus"
-	GrafanaComponentName       = "grafana"
-	KyvernoComponentName       = "kyverno"
-	KyvernoPolicyComponentName = "kyverno-policies"
-	VeleroComponentName        = "velero"
-	RookOperatorComponentName  = "rook"
-	RookClusterComponentName   = "rook-ceph"
-	FlaggerComponentName       = "flagger"
-	TestloaderComponentName    = "testloader"
+	MetricPluginName             = "metric"
+	GrafanaPluginName            = "grafana"
+	KyvernoPluginName            = "kyverno"
+	BackupPluginName             = "backup"
+	StorageOperatorPluginName    = "storage-operator"
+	ClusterStoragePluginName     = "cluster-storage"
+	FlaggerPluginName            = "flagger"
+	PublicTestloaderName         = "testloader"
+	SubMarinerBrokerPluginName   = "submariner-broker"
+	SubMarinerOperatorPluginName = "submariner-operator"
+
+	ThanosComponentName             = "thanos"
+	PrometheusComponentName         = "prometheus"
+	GrafanaComponentName            = "grafana"
+	KyvernoComponentName            = "kyverno"
+	KyvernoPolicyComponentName      = "kyverno-policies"
+	VeleroComponentName             = "velero"
+	RookOperatorComponentName       = "rook"
+	RookClusterComponentName        = "rook-ceph"
+	FlaggerComponentName            = "flagger"
+	TestloaderComponentName         = "testloader"
+	SubMarinerBrokerComponentName   = "sm-broker"
```

Comment:
please use the full name(submariner) if there's no particular reason

Reply:
This abbr is due to the fact that helm has a [limit on name of release](https://github.com/helm/helm/blob/109dbe7c18a550e663f616e0d516558855706753/pkg/chartutil/validate_name.go#L44) (mentioned in [Helm#6006](https://github.com/helm/helm/issues/6006)). I will try to revalidate if full name will cause the too long name excepetion.

Reply:
To be more specific, when `SubMarinerBrokerComponentName`  is  "submariner-broker", broker cannot be installed because `helm-controller reconciliation failed: Helm install failed: release name "submariner-k8s-broker-submariner-broker-kurator-member1`